### PR TITLE
modify html and css

### PR DESCRIPTION
**Pull request by CELYNEBIRUNGI**
## Changes
- Added basic HTML & CSS structure for my portfolio
- Added home page for my portfolio
- Modify the README template to my personal information.
- Modified the css and html file
-Checked the  requirements of the project
Updated the css and html file
Added more html semantics

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,7 +1,6 @@
 <!DOCTYPE html>
```
Comment (AbelG101) [inline index.html:1]: - Kindly include the proper font used in the [Figma design](https://github.com/musangamfure/portfolio/pull/1/files#diff-0eb547304658805aad788d320f10bf1f292797b5e6d745a3bf617584da017051R40-R62)🎨 as this ensures you can properly follow a given design to your future clients.
    - The font is called "Poppins" and you can find it [here](https://fonts.google.com/specimen/Poppins?query=poppi&preview.text=I%E2%80%99m%20Glad%20to%20see%20you!&preview.text_type=custom)

File: index.html [hunk 2]
```diff
@@ -10,15 +9,15 @@
...
         <nav id="nav-hamburger">
-            <img src="/images/nav.png">        
+        <img src="/images/nav.png">        
         </nav>
     </header>
     <section id="main">
-        <h1 class="main-heading">I'm Celyne.<br> Glad to see you!</h1>
+        <h1 class="main-heading">I'm Evelyne Celyne<br> Glad to see you!</h1>
         <p class="main-paragraph">I'm a software developer! I can help you build a product, feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don't hesitate to contact me.</p>
         <label class="connect">LET'S CONNECT</label>
```
Comment (AbelG101) [inline index.html]: - [ ] Please ensure the "LET'S CONNECT" is either in a link tag like the `a` tag or in a `button` element.

File: index.html [hunk 3]
```diff
@@ -29,8 +28,11 @@ <h1 class="main-heading">I'm Celyne.<br> Glad to see you!</h1>
           <li><a href="#" class="icons"><img src="/images/assets/github.png"></a></li>
           <li><a href="#" class="icons"><img src="/images/assets/hi 5.png"></a></li>
         </ul><br>
+        <button type="button">Get my resume</button>
```
Comment (AbelG101) [inline index.html]: - Kindly remove this button as it's not specified in the [Figma design](https://github.com/musangamfure/portfolio/pull/1/files#diff-0eb547304658805aad788d320f10bf1f292797b5e6d745a3bf617584da017051R40-R62) you should try to stay consistent with the design as much as possible. ✔

File: index.html [hunk 4]
```diff
@@ -29,8 +28,11 @@ <h1 class="main-heading">I'm Celyne.<br> Glad to see you!</h1>
           <li><a href="#" class="icons"><img src="/images/assets/github.png"></a></li>
           <li><a href="#" class="icons"><img src="/images/assets/hi 5.png"></a></li>
         </ul><br>
+        <button type="button">Get my resume</button> 
+  
         </div>
       </section>
```
Comment (AbelG101) [inline index.html:38]: - Please make sure you add the proper padding and spacing for the main section here as shown in the design
      ApplicationFrameHost_TgYJFZFGqZ

- I've highlighted the left and right padding you seem to be missing and kindly add `100vh` as the height of this section as per the [Figma design](https://github.com/musangamfure/portfolio/pull/1/files#diff-0eb547304658805aad788d320f10bf1f292797b5e6d745a3bf617584da017051R40-R62).
     Inkedfirefox_idvdmwbhjL

**Review by AbelG101 — CHANGES_REQUESTED** [review]
# Hi @CELYNEBIRUNGI 👋, 

You're doing a good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- Correct use of GitHub Flow ✔
-  Well-structured files ✔
- All linter checks are passing ✔
- Descriptive PR title with a great summary ✔

## Required Changes ♻️
- _Check the comments under the review._
- You're going well here just a few changes and you'll be done 🚀
## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.css [hunk 5]
```diff
@@ -1,8 +1,16 @@
+@import url('https://fonts.googleapis.com/css2?family=Inter:wght@100;200;300;400;500;600;700;800;900&display=swap');
```
Comment (braincee) [inline index.css:1]: - [ ] Please I highly suggest sticky to the [Figma Design] as mentioned by the previous reviewer. 😄 

**REQUIRED FIGMA DESIGN**

Screenshot (73)


_**YOUR DESIGN**_

Screenshot (99)

File: index.css [hunk 6]
```diff
@@ -1,8 +1,16 @@
...
-  margin: 0;
-  background-color: #e5e5e5;
+  background-color: white;
+  height: fit-content;
+  border-radius: 16px;
+  border: 1px solid #dfe1e6;
+  box-shadow: 5px 10px;
 }
 
 .header {
```
Comment (braincee) [inline index.css:17]: - [ ] Good job implementing the CSS Flexbox in the `header` class. I suggest you change the `header` class to the `navbar` class to follow HTML semantic tag requirements. 😄

File: index.css [hunk 7]
```diff
@@ -60,10 +68,11 @@ body {
 
 #main {
```
Comment (braincee) [inline index.css:76]: - [ ] Good job implementing CSS Flexbox in the `main` content. I suggest changing it to a `section` tag to follow HTML semantic tag requirements. 😄

File: index.css [hunk 8]
```diff
@@ -91,6 +100,37 @@ body {
...
+  padding-left: 2%;
+  margin-bottom: 0.5%;
+  color: #344563;
+  font-weight: 7%;
+  font-style: normal;
+  font-size: 75%;
+  line-height: 24px;
+}
+
 @media screen and (min-width: 408px) {
```
Comment (braincee) [inline index.css]: - [ ] Good job implementing responsive breakpoints. I suggest using `375px` for the mobile version minimum and `768px` for the desktop version minimum per the project requirements. 😃

File: index.css [hunk 9]
```diff
@@ -91,6 +100,37 @@ body {
...
+  line-height: 24px;
+  color: #7b86ece6;
+  font-style: normal;
+  padding-left: 10%;
+  margin-top: 2%;
+}
+
+.main-heading {
+  padding-left: 75%;
+  padding-right: 12%;
```
Comment (braincee) [inline index.css]: - [ ] Good job implementing responsive values in your CSS rules per project requirements. 👍

**Review by braincee — CHANGES_REQUESTED** [review]
Hello @CELYNEBIRUNGI  ✋, I am @braincee your Code Reviewer

Great work on making the changes 👏 
You've done well implementing some of the requested changes, but there are still some which aren't been addressed yet.


so-close

HIGHLIGHTS
------

 - GitHub flow rightfully used ✅
 - Meaningful title and summary for PR ✅
 - Professional README.md file ✅
 - No linter errors ✅

REQUIRED CHANGES ♻️
------
**_Please check the comments under the review._**

OPTIONAL CHANGES
--------
**_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._**

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @braincee in your question so I can receive the notification.

_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._

----

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**